packages/eslint-plugin-next/lib/index.js

```diff
@@ -39,12 +39,12 @@ module.exports = {
         '@next/next/next-script-for-ga': 1,
         '@next/next/no-document-import-in-page': 2,
         '@next/next/no-head-import-in-document': 2,
-        '@next/next/no-script-in-document': 2,
         '@next/next/no-script-component-in-head': 2,
         '@next/next/no-server-import-in-page': 2,
         '@next/next/no-typos': 1,
         '@next/next/no-duplicate-head': 2,
         '@next/next/inline-script-id': 2,
+        '@next/next/no-before-interactive-script-outside-document': 2,
```

Comment:
Can we only apply this change/lint rule when using streaming to prevent a breaking change and only require users to migrate their `beforeInteractive` scripts when migrating to streaming?

Reply:
Users, don't have to migrate for their apps to work. This change is backward compatible. We want to start throwing warnings now and in a future release make it a breaking change.